projectDocs/translating/github.md

```diff
@@ -0,0 +1,58 @@
+- `gestures.ini`: [Translating Gestures](https://www.nvaccess.org/files/nvda/documentation/developerGuide.html#TranslatingGestures)
+
+To keep these files up to date, translators must be concious of changes in NVDA.
+For changes to `symbols.dic` and `characterDescriptions.dic`, please subscribe to the read-only [nvda-l10n](https://groups.google.com/a/nvaccess.org/g/nvda-l10n) mailing list.
+When changes to these files in English occur, you will be notified via this mailing list.
+This allows you to consider adding equivalent descriptions to your localised file.
+
+For changes to NVDA's input gestures, please refer to the [latest changes in NVDA](../../user_docs/en/changes.md).
+New gestures will be announced there, and localisations can be added to `gestures.ini` if required.
+
+## Process
+
+1. Find the relevant file to edit in the GitHub directory
+    - Visit [source/locale](https://github.com/nvaccess/nvda/tree/beta/source/locale)
+    - Open the directory with your language code
+        - English example: <https://github.com/nvaccess/nvda/tree/beta/source/locale/en>
+        - Format (replace `{lang}`): `https://github.com/nvaccess/nvda/tree/beta/source/locale/{lang}`
+    - Find the relevant file:
+        - `symbols.dic`, `characterDescriptions.dic` or `gestures.ini`
+        - English example: <https://github.com/nvaccess/nvda/blob/beta/source/locale/en/symbols.dic>
+        - Format (replace `{lang}` and `{fileName}`): `https://github.com/nvaccess/nvda/blob/beta/source/locale/{lang}/{fileName}`
+1. If the relevant file doesn't exist yet, create a new one.
+    - To create a new file use the "add file" button from your language's directory.
+        - English example: <https://github.com/nvaccess/nvda/new/beta/source/locale/en>
+        - Format (replace `{lang}`): `https://github.com/nvaccess/nvda/new/beta/source/locale/{lang}`
+    - `characterDescriptions.dic`: Copy and paste desired contents from the [English example](https://raw.githubusercontent.com/nvaccess/nvda/refs/heads/beta/source/locale/en/characterDescriptions.dic).
+    Note you should translate all the content you copy.
+    - `symbols.dic`: Copy and paste desired contents from the [English example](https://raw.githubusercontent.com/nvaccess/nvda/refs/heads/beta/source/locale/en/symbols.dic).
+    Note you should translate all the content you copy.
+    - `gestures.ini`: This file doesn't require a base file, gestures can be added as needed.
+1. Edit the relevant file
+    - Tab to the "edit file" button to open the editor for the file.
+    - English example: <https://github.com/nvaccess/nvda/edit/beta/source/locale/en/symbols.dic>
+    - Format (replace `{lang}` and `{fileName}`): `https://github.com/nvaccess/nvda/edit/beta/source/locale/{lang}/{fileName}`
+    - Refer to [background](#background) for more information on the format of the files
+1. Submit your changes
+    - Press the "commit changes" button.
```

Comment:
Sometimes I edit files from GitHub, but I don't find the commit button if I don't press e `control+s`.
If it is not needed, ignore the following suggestion. 

```suggestion
    - Press `control+s` to save changes
    - Press the "commit changes" button.
```

Reply:
@nvdaes - it seems these suggestions are equivalent - e.g. `ctrl+s` is the same as pressing the "commit changes" button, they both trigger the commit dialog.

```suggestion
    - Press `control+s` or the "commit changes" button to open a dialog to save and propose the changes.
```

Reply:
OK, understood